src/Taxjar/DependencyInjection/TaxjarOptions.cs

```diff
@@ -0,0 +1,25 @@
+#if NETSTANDARD2_0
+using System;
+
+namespace Taxjar.Options
+{
+    public class TaxjarOptions
+    {
+
+        /// <summary>
+        /// Taxjar token.
+        /// </summary>
+        public string ApiToken { get; set; }
+
+        /// <summary>
+        /// Is Sandbox?
+        /// </summary>
+        public bool IsSandBox { get; set; }
+
+        /// <summary>
+        /// Timeout for the HttpClient.
+        /// </summary>
+        public TimeSpan Timeout { get; set; }
```

Comment:
Prefer to use `int` directly here, unless you have a compelling case for usage of `TimeSpan`.

Reply:
What I have experienced from .net side of things it is easier to add a timestamp rather than figure out if it is seconds, min, hour... etc.